common/CMakeLists.txt

```diff
@@ -1,766 +1,772 @@
-	repositories/character_activities_repository.h
-	repositories/character_alternate_abilities_repository.h
-	repositories/character_alt_currency_repository.h
-	repositories/character_auras_repository.h
-	repositories/character_bandolier_repository.h
-	repositories/character_bind_repository.h
-	repositories/character_buffs_repository.h
-	repositories/character_corpses_repository.h
-	repositories/character_corpse_items_repository.h
-	repositories/character_currency_repository.h
-	repositories/character_data_repository.h
-	repositories/character_disciplines_repository.h
-	repositories/character_expedition_lockouts_repository.h
-	repositories/character_inspect_messages_repository.h
-	repositories/character_instance_safereturns_repository.h
-	repositories/character_item_recast_repository.h
-	repositories/character_languages_repository.h
-	repositories/character_leadership_abilities_repository.h
-	repositories/character_material_repository.h
-	repositories/character_memmed_spells_repository.h
-	repositories/character_pet_buffs_repository.h
-	repositories/character_pet_info_repository.h
-	repositories/character_pet_inventory_repository.h
-	repositories/character_potionbelt_repository.h
-	repositories/character_skills_repository.h
-	repositories/character_spells_repository.h
-	repositories/character_task_timers_repository.h
-	repositories/character_tasks_repository.h
-	repositories/char_create_combinations_repository.h
-	repositories/char_create_point_allocations_repository.h
-	repositories/char_recipe_list_repository.h
-	repositories/completed_tasks_repository.h
-	repositories/completed_shared_tasks_repository.h
-	repositories/completed_shared_task_members_repository.h
-	repositories/completed_shared_task_activity_state_repository.h
-	repositories/content_flags_repository.h
-	repositories/damageshieldtypes_repository.h
-	repositories/data_buckets_repository.h
-	repositories/db_str_repository.h
-	repositories/discovered_items_repository.h
-	repositories/doors_repository.h
-	repositories/dynamic_zones_repository.h
-	repositories/dynamic_zone_members_repository.h
-	repositories/dynamic_zone_templates_repository.h
-	repositories/eventlog_repository.h
-	repositories/expeditions_repository.h
-	repositories/expedition_lockouts_repository.h
-	repositories/faction_base_data_repository.h
-	repositories/faction_list_repository.h
-	repositories/faction_list_mod_repository.h
-	repositories/faction_values_repository.h
-	repositories/fishing_repository.h
-	repositories/forage_repository.h
-	repositories/friends_repository.h
-	repositories/global_loot_repository.h
-	repositories/gm_ips_repository.h
-	repositories/goallists_repository.h
-	repositories/graveyard_repository.h
-	repositories/grid_repository.h
-	repositories/grid_entries_repository.h
-	repositories/ground_spawns_repository.h
-	repositories/group_id_repository.h
-	repositories/group_leaders_repository.h
-	repositories/guilds_repository.h
-	repositories/guild_members_repository.h
```

Comment:
`repositories/guild_members_repository.h` removed and not added back

Reply:
Table probably doesn't follow current criteria for generation is my guess

Reply:
```Table primary key [guild_members] has no primary key! Skipping... ```